recipes/_generate_password.rb

```diff
@@ -10,7 +10,12 @@
 
 require 'securerandom'
 
-password_file = File.join node['splunk']['external_config_directory'], 'password'
+# For a node that has multiple splunk instances, create a separate password file.
+password_file = if node['splunk']['ignore_already_installed_instance'] == true
```

Comment:
minor, shouldn't need the `== true` here.  I'm surprised rubocoop didn't flag that.

Reply:
Removed: https://github.com/cerner/cerner_splunk/pull/265/commits/b82406d1b35f6d5af0f356b65b48a4bad25c628b

Reran rubocop as a sanity check, still didn't catch it.